plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -377,6 +382,33 @@ public DeltaLakeConfig setPerTransactionMetastoreCacheMaximumSize(long perTransa
         return this;
     }
 
+    public boolean isStoreTableMetadataEnabled()
+    {
+        return storeTableMetadataEnabled;
+    }
+
+    @Config("delta.metastore.store-table-metadata-enabled")
+    @ConfigDescription("Store table definition to metastore")
+    public DeltaLakeConfig setStoreTableMetadataEnabled(boolean storeTableMetadataEnabled)
+    {
+        this.storeTableMetadataEnabled = storeTableMetadataEnabled;
+        return this;
+    }
+
+    @Min(0)
```

Comment:
`Min(0)` feels strange for the number of threads.
`delta.metastore.store-table-metadata-threads` is optional - i'd say either a positive number or empty